lldb/source/Core/Debugger.cpp

```diff
@@ -761,12 +767,29 @@ void Debugger::InstanceInitialize() {
 
 DebuggerSP Debugger::CreateInstance(lldb::LogOutputCallback log_callback,
                                     void *baton) {
+#ifdef LLVM_BUILD_TELEMETRY
+  lldb_private::telemetry::SteadyTimePoint start_time =
+      std::chrono::steady_clock::now();
+#endif
```

Comment:
I think we should wrap this into an RAII object (like Progress) so that this does the right thing automatically.

Reply:
(NOT done - this is pending the result of whether we'd revert the ifdef approach and instead just make the cmake variable control llvm::telemetry::Config::EnableTelemetry)